pjsip4net/Calls/Call.cs

```diff
@@ -277,7 +277,7 @@ public void Transfer(string destinationUri)
             GuardDisposed();
             Helper.GuardNotNullStr(destinationUri);
             Helper.GuardIsTrue(new SipUriParser(destinationUri).IsValid);
-            if (_mediaSession.IsActive)
+            if (_mediaSession.IsActive || this._mediaSession.MediaState == CallMediaState.None)

```

Comment:
According to the pjsua code there should be no checks regarding a call state: https://trac.pjsip.org/repos/browser/pjproject/trunk/pjsip/src/pjsua-lib/pjsua_call.c#L2784
Let's remove this line completely.